mewarpx/examples/capacitive_discharge.py

```diff
@@ -93,7 +93,7 @@
 #    grid=mwxrun.grid, method='Multigrid', required_precision=1e-12
 # )
 solver = poisson_pseudo_1d.PoissonSolverPseudo1D(grid=mwxrun.grid)
-
+mwxrun.simulation.external_solver = True
```

Comment:
Same comment as on Slack, just wanted to put it here for easy reference.
I think for the sake of simplicity instead of adding the line `mwxrun.simulation.external_solver = True` to input files, you should add a line to `poisson_pseudo_1d.PoissonSolverPseudo1D()` in the `__init__()` function to set the external solver parameter to True.